clouddesigner/org.occiware.clouddesigner.occi.pca/META-INF/MANIFEST.MF

```diff
@@ -1,17 +1,6 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
-Bundle-Name: OCCIware ProActive Cloud Automation OCCI Extension
+Bundle-Name: Procci
 Bundle-SymbolicName: org.occiware.clouddesigner.occi.pca;singleton:=true
-Bundle-Version: 0.1.0.qualifier
-Bundle-ClassPath: .
-Bundle-Vendor: OCCIware
-Bundle-RequiredExecutionEnvironment: JavaSE-1.7
-TODO-Export-Package: org.occiware.clouddesigner.occi.pca,
- org.occiware.clouddesigner.occi.pca.impl,
- org.occiware.clouddesigner.occi.pca.util
-Require-Bundle: org.eclipse.core.runtime,
- org.eclipse.emf.ecore;visibility:=reexport,
- org.eclipse.ocl.pivot;visibility:=reexport,
- org.occiware.clouddesigner.occi.infrastructure;visibility:=reexport,
- org.occiware.clouddesigner.occi;visibility:=reexport
-Bundle-ActivationPolicy: lazy
+Bundle-Version: 1.0.0.qualifier
```

Comment:
This does not match the version specified in the POM.
It explains why the build is broken.

As a reminder, your POM indicates `0.1.0-SNAPSHOT`.
So, your bundle version should be `0.1.0.qualifier`. Or you should update your Maven version otherwise.